### DISCUSSION TITLE
“Validation” button not appearing during Manage SLC’s directory

### DISCUSSION THREAD

**Discussion by junaidlonez11** [alexisInSAR/EZ-InSAR#83] [Help]
Hi,

After Clicking “Manage SLC’s directory” --> Opens a dialog showing the paths of the “slc”, “dem”, and “aux” folders.
The text of the paths is in green color, thus the folders are successfully detected. However, the “Validation” button does not appear to finish the checking of the paths.

![image](https://github.com/alexisInSAR/EZ-InSAR/assets/57167285/22b93802-7b91-46c3-84ef-ecafd52d8ffb)

Thanks!

**Comment by alexisInSAR** [comment]
Hi @junaidlonez11, 

There is not a "validation" button, saving is automatic. If the paths are green, you can close the window. 

Kind regards, 

Alexis

**Reply by junaidlonez11** [reply]
Thanks, I noticed that later! I was trying to follow the tutorial as provided and there validation button was showing up.
